acceptance/acceptance_test.go

```diff
@@ -211,13 +211,8 @@ func doComparison(t *testing.T, pathExpected, pathNew, valueNew string) {
 	valueExpected = testdiff.NormalizeNewlines(valueExpected)
 	testdiff.AssertEqualTexts(t, pathExpected, pathNew, valueExpected, valueNew)
 	if testdiff.OverwriteMode {
-		if valueNew != "" {
-			t.Logf("Overwriting: %s", pathExpected)
-			testutil.WriteFile(t, pathExpected, valueNew)
-		} else {
-			t.Logf("Removing: %s", pathExpected)
```

Comment:
Why remove this? It's helpful to update golden test files when files are deleted.

Reply:
Auto-removal functionality is quite helpful actually. I'd prefer that we keep it.

We can do so by changing readIfExists to return (contents string, exists bool) to distinguish between empty and missing.

Reply:
I missed this nuance, thanks.

I updated the code to deal with all combinations of missing files in a single place.

Please re-review the changes to `acceptance_test.go`.